lib/api-config.rb

```diff
@@ -66,13 +66,11 @@ def configuration_for(which)
       end
 
       def parse(source)
-        config =
-          if YAML.respond_to?(:unsafe_load_file)
-            YAML.unsafe_load_file(source).fetch(APIConfig.env)
-          else
-            YAML.load_file(source).fetch(APIConfig.env)
-          end
-
+        config = if YAML.respond_to?(:unsafe_load)
+                   YAML.unsafe_load(ERB.new(File.read(source)).result).fetch(APIConfig.env)
```

Comment:
can we move `ERB.new(File.read(source)).result` to a different variable and preserve the old indentation?

Reply:
@ajgon  make it prettier